MeshLib/VtkOGSEnum.cpp

```diff
@@ -92,7 +92,7 @@ int OGSToVtkCellType(CellType ogs)
 			return VTK_PYRAMID;
 		case CellType::PYRAMID13:
 			return VTK_QUADRATIC_PYRAMID;
-		case CellType::INVALID:
+		default:
```

Comment:
why not add macros to include a default case only when MSVC is used? we can still get warnings with gcc and clangs if one would add an entry to the CellType enum.